### PR TITLE
feat: add cli util to generate new components

### PR DESCRIPTION
**Pull request by kikoruiz**
## FEAT: add cli util to generate new components

**TASK**: #205 

### Description, Motivation and Context
Have available a CLI tool to generate new components. The command suggested is the following one: `npx spark g button -D "My fancy button"`

### Types of changes

- [x] 🛠️ Tool

### CODE REVIEW

File: packages/utils/cli/templates/stories/[Component.stories.tsx].js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { pascalCase } from 'pascal-case'
```
Comment (acd02) [inline packages/utils/cli/templates/stories/[Component.stories.tsx].js:1]: We don't need the `stories` folder anymore 🧹 🥳

**Review by acd02 — APPROVED** [review]

File: packages/utils/cli/templates/stories/[Component.stories.tsx].js (hunk 1 above)
Comment (Powerplex) [reply packages/utils/cli/templates/stories/[Component.stories.tsx].js:1]: Indeed, package structure is now documented here and everything is flat inside the `src` folder: https://sparkui.vercel.app/?path=/docs/definitions-package-structure--docs

File: packages/utils/cli/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@spark-ui/cli-utils",
```
Comment (Powerplex) [inline packages/utils/cli/package.json:2]: I am wondering if we should not dedicate this package to scaffolding other packages and rename it to something less generic ?

Comment (kikoruiz) [reply packages/utils/cli/package.json:2]: I think it's nice to have something global like `spark [command]`, we should add each command in the package.json config. What do you think?

File: packages/utils/cli/templates/stories/[Component.stories.tsx].js (hunk 1 above)
Comment (kikoruiz) [reply packages/utils/cli/templates/stories/[Component.stories.tsx].js:1]: Nice!! I see that it's working fine right now 🥳

File: packages/utils/cli/package.json (hunk 2 above)
Comment (andresin87) [reply packages/utils/cli/package.json:2]: What if there is another directory for tool packages called `tools` like `hooks` and `components`. Maybe not initially but if we consider to become an standard in the company this tool will be useful to create brand components following the same pattern

Comment (Powerplex) [reply packages/utils/cli/package.json:2]: Regarding directory (workspace) we can always update later it won't impact the package name that is published :) 

(What I mean is we can always change it later if it is merged as is)

File: packages/utils/cli/bin/spark-generate.mjs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          .replace('Component', pascalCase(component))
+          .replaceAll(/\[|\]|\.js$/g, ''),
+        content: module.default({ component, description }),
+      }))
+    )
+
+    const filesToWrite = await Promise.all(templateContents)
+
+    Promise.all(filesToWrite.map(writeFile)).then(() => {
+      log('All component files has been properly written!')
```
Comment (kikoruiz) [inline packages/utils/cli/bin/spark-generate.mjs:53]:
```suggestion
      log('All component files have been properly written!')
```

**Pull request merged**